src/domain/request/port/RequestRepositoryPort.spec.js

```diff
@@ -0,0 +1,16 @@
+const { describe, it, beforeEach } = require('mocha');
+const { expect } = require('chai');
+const RequestRepositoryPort = require('./RequestRepositoryPort');
+
+describe('RequestRepositoryPort', () => {
```

Comment:
Because RequestRepositoryPort is a class due to javascript limitations it lowers code coverage percentage. It gives a value of cleaner view where there aren't enough tests constantly skipping all ports coverage.